### PR TITLE
Pulidoc

### PR DESCRIPTION
**Pull request by pulidoc**
I would like for my code to be pulled in.

### CODE REVIEW

**Review by fairsilas — CHANGES_REQUESTED** [review]
Get rid of hw03_pulidoc.py. That file is incorrectly named. I can only merge a03_pulidoc.py

**Comment by pulidoc** [comment]
I changed the name.

File: a03_pulidoc.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    petals.color("#67AC2B")
+
+    create_roof(circle)      # Function call to function_1
+    create_house(bunny)  # Function call to function_2
+    create_window(dolphin)  # Function to call to function_3
+    create_windows(oso)     # function to call
+    make_door(tiger)        # function to call
+    make_flower(flower)     # function to call
+    make_other(petals)      #fucntion to call
+
```
Comment (sheggen) [inline a03_pulidoc.py:190]: you're missing a `wn.exitonclick()` at the end of main.

File: hw03_pulidoc.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
+
```
Comment (sheggen) [inline hw03_pulidoc.py:1]: Why two files?? Pick one and delete the other. They look exactly the same.

Comment (pulidoc) [reply hw03_pulidoc.py]: I believe I deleted it

File: a03_pulidoc.py (hunk 1 above)
Comment (pulidoc) [reply a03_pulidoc.py:192]: fixed. Thank you

**Pull request merged**